sia-react-native/navigation/SwitchNavigator.js

```diff
@@ -19,6 +22,12 @@ const SwitchNavigator = createSwitchNavigator(
         Profile: {
             screen: Profile
         },
+        list: {
+            screen: EventListFront
+        },
+        form: {
+            screen: EventForm
+        },
```

Comment:
Minor but change to List and Form so we can stay consistent with the format we have been using.

Reply:
No problem